apps/website/src/examples/Button.main.tsx

```diff
@@ -3,40 +3,32 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 import * as React from 'react';
-import { Button } from '@itwin/itwinui-react';
+import { Button, SplitButton, MenuItem, IconButton, IdeasButton } from '@itwin/itwinui-react';
+import { SvgPlaceholder } from '@itwin/itwinui-icons-react';
 
 export default () => {
+  const menuItems = (close: () => void) => [
+    <MenuItem key={1} onClick={close}>
+      Item #1
+    </MenuItem>,
+    <MenuItem key={2} onClick={close}>
+      Item #2
+    </MenuItem>,
+  ];
+
   return (
     <div>
-      <div style={{ display: 'flex', gap: '24px', flexWrap: 'wrap', margin: '12px' }}>
-        <Button size='small'>Default</Button>
```

Comment:
I feel the current sandbox example is more intuitive than the changed one. Since the props table below is only for `Button` and not for `SplitButton` or `IconsButton`, displaying `SplitButton` and `IconButton` in the same sandbox can be confusing.

Reply:
Combined docs for buttons, showing different examples for different buttons.